be/src/common/status.h

```diff
@@ -151,8 +155,18 @@ class Status {
         return Status(TStatusCode::DATA_QUALITY_ERROR, msg, precise_code, msg2);
     }
 
+    // A wrapper for OLAPStatus
+    //      Precise code is for OLAPStatus's enum value
+    //      All Status Error is treated as Internal Error 
+    static Status OLAPInternalError(int16_t precise_code, const Slice& msg = Slice()) {
+        #ifdef PRINT_ALL_ERR_STATUS_STACKTRACE
+        LOG(WARNING) << "Error occurred, error code = " << precise_code << ", with message: " << msg
+                     << "\n" << boost::stacktrace::stacktrace();
```

Comment:
Can the stacktrace be printed for all errors, but the same stacktrace will not be printed after printing several times, like JAVA, until the next restart of BE.

Reply:
Does java have this feature? Could you show me some demo code in JAVA?

Reply:
Search jvm option `OmitStackTraceInFastThrow`

Reply:
This is an optimization of HotSpot VM called fast throw, I'm not sure if C++ has a similar optimization. Or can we do it ourselves.

Ref: https://stackoverflow.com/questions/42115284/what-are-jvm-pre-allocated-exceptions
        https://www.jianshu.com/p/e87d166380eb

Reply:
@xinyiZzz Currently, I won't implement this feature in Doris, because it depends on unify the error code work.
 I will do following task after this PR is closed:
1. Replace OLAPStatus with Status
2. Unify ErrorCode, for example TStatusCode, OLAPStatus, PStatus, I will unify them to only one error code
3. Do some statistics on ErrorCodes, for example, we could find how many IO Error happens.
4. After all these 3 tasks are finished, I think maybe we could implement OmitStackTraceInFastThrow. But I am not sure because it will ignore some errors.

Reply:
Great, looking forward.